src/WebApi.OutputCache.V2/CacheOutputHttpVerbsAttribute.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Web.Http.Filters;
+
+namespace WebApi.OutputCache.V2
+{
+	/// <summary>
+	/// When this attribute is used, only the specified vebrs will participate in cache.
+	/// Otherwise GET method only.
+	/// </summary>
+	[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = true, Inherited = true)]
+	public class CacheOutputHttpVerbsAttribute : ActionFilterAttribute
+	{
+		/// <summary>
+		/// Gets or sets the list of HTTP verbs that needs to participate in cache.
+		/// </summary>
+		public string[] Verbs { get; private set; }
```

Comment:
Unfortunately no. Only simple types may be used on attribute e.g [CacheOutputHttpVerbs(HttpMethod.Get)] - produces a compilation error.